src/Utilities/MoistThermodynamics/src/MoistThermodynamics.jl

```diff
@@ -363,31 +390,30 @@ end
 """
     phase_partitioning_eq!(q_l, q_i, T, ρ, q_t)
 
-Return the partitioning of the phases in equilibrium.
+    # ASR Do we need this function at all(?)
+
```

Comment:
We had the wrapper to roll out the functions over array elements because it was unclear how to handle broadcasting for functions with multiple output arguments. See issue #41. Perhaps @simonbyrne has suggestions for how to do this better?

Reply:
I had a similar situation in RootSolvers. @simonbyrne 's improvement there was simply to broadcast [`first`](https://github.com/climate-machine/CLIMA/pull/117/files#diff-439cbcd21af5d189648a0a476d3b94fcR118), which I think is a nice solution. Based on this, all we need is the element-wise function, and we no longer need the wrapper function.

Reply:
Great. @asraero , could you make the corresponding change in MoistThermodynamics (and elsewhere where needed)?